feature_engine/selection/drop_psi_features.py

```diff
@@ -7,42 +7,31 @@
 from pandas.api.types import is_numeric_dtype
 
 from feature_engine._docstrings.fit_attributes import (
-    _feature_names_in_docstring,
-    _n_features_in_docstring,
-)
-from feature_engine._docstrings.init_parameters.selection import (
-    _confirm_variables_docstring,
-)
+    _feature_names_in_docstring, _n_features_in_docstring)
+from feature_engine._docstrings.init_parameters.selection import \
+    _confirm_variables_docstring
 from feature_engine._docstrings.methods import _fit_transform_docstring
 from feature_engine._docstrings.substitute import Substitution
-from feature_engine.dataframe_checks import (
-    _check_contains_inf,
-    _check_contains_na,
-    check_X,
-)
-from feature_engine.discretisation import (
-    EqualFrequencyDiscretiser,
-    EqualWidthDiscretiser,
-)
+from feature_engine.dataframe_checks import (_check_contains_inf,
+                                             _check_contains_na, check_X)
+from feature_engine.discretisation import (EqualFrequencyDiscretiser,
+                                           EqualWidthDiscretiser)
 from feature_engine.selection._docstring import (
-    _get_support_docstring,
-    _variables_attribute_docstring,
-    _variables_numerical_docstring,
-)
+    _get_support_docstring, _variables_all_docstring,
+    _variables_attribute_docstring)
 from feature_engine.selection.base_selector import BaseSelector
-from feature_engine.variable_handling._init_parameter_checks import (
-    _check_init_parameter_variables,
-)
-from feature_engine.variable_handling.variable_type_selection import (
-    find_or_check_numerical_variables,
-)
+from feature_engine.tags import _return_tags
+from feature_engine.variable_handling._init_parameter_checks import \
+    _check_init_parameter_variables
+from feature_engine.variable_handling.variable_type_selection import \
+    find_categorical_and_numerical_variables
```

Comment:
Could you format this with `black`?

```
black feature_engine/selection/drop_psi_features.py
```

You should first run `isort`, then `black`. Not the other way around

Reply:
Yep, I second this one. Not sure what happened here, but if there are no new imports, I'd prefer we don't change what was originally there. It is also easier to review.

Could we fix @dlaprins ?

Reply:
Thanks for letting me know,  I was unaware of the correct ordering.